src/test/java/org/broadinstitute/hellbender/tools/copynumber/plotting/PlotDenoisedCopyRatiosIntegrationTest.java

```diff
@@ -62,13 +49,44 @@ public void testPlotting() {
         Assert.assertEquals(scaledDeltaMAD, (standardizedMAD - denoisedMAD) / standardizedMAD);
     }
 
+    //checks that output files with reasonable file sizes are generated, but correctness of output is not checked
+    @Test(groups = "R")
+    public void testPlotting() {
+        final File outputDir = createTempDir("testDir");
+        final String[] arguments = {
+                "--" + CopyNumberStandardArgument.STANDARDIZED_COPY_RATIOS_FILE_LONG_NAME, STANDARDIZED_COPY_RATIOS_FILE.getAbsolutePath(),
+                "--" + CopyNumberStandardArgument.DENOISED_COPY_RATIOS_FILE_LONG_NAME, DENOISED_COPY_RATIOS_FILE.getAbsolutePath(),
+                "--" + StandardArgumentDefinitions.SEQUENCE_DICTIONARY_NAME, SEQUENCE_DICTIONARY_FILE.getAbsolutePath(),
+                "-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME, outputDir.getAbsolutePath(),
+                "--" + CopyNumberStandardArgument.OUTPUT_PREFIX_LONG_NAME, OUTPUT_PREFIX
+        };
+        runCommandLine(arguments);
+        assertFilesCreated(outputDir);
+    }
+
+    //checks that output files with reasonable file sizes are generated, but correctness of output is not checked
+    @Test(groups = "R")
+    public void testPlottingInfiniteMaximumCopyRatio() {
+        final File outputDir = createTempDir("testDir");
+        final String[] arguments = {
+                "--" + CopyNumberStandardArgument.STANDARDIZED_COPY_RATIOS_FILE_LONG_NAME, STANDARDIZED_COPY_RATIOS_FILE.getAbsolutePath(),
+                "--" + CopyNumberStandardArgument.DENOISED_COPY_RATIOS_FILE_LONG_NAME, DENOISED_COPY_RATIOS_FILE.getAbsolutePath(),
+                "--" + StandardArgumentDefinitions.SEQUENCE_DICTIONARY_NAME, SEQUENCE_DICTIONARY_FILE.getAbsolutePath(),
+                "--" + PlottingUtils.MAXIMUM_COPY_RATIO_LONG_NAME, "Infinity",
+                "-" + StandardArgumentDefinitions.OUTPUT_SHORT_NAME, outputDir.getAbsolutePath(),
+                "--" + CopyNumberStandardArgument.OUTPUT_PREFIX_LONG_NAME, OUTPUT_PREFIX
+        };
+        runCommandLine(arguments);
+        assertFilesCreated(outputDir);
+    }
+
     @Test(expectedExceptions = IllegalArgumentException.class)
     public void testMinimumContigLength() {
         final File outputDir = createTempDir("testDir");
         final String[] arguments = {
                 "--" + CopyNumberStandardArgument.STANDARDIZED_COPY_RATIOS_FILE_LONG_NAME, STANDARDIZED_COPY_RATIOS_FILE.getAbsolutePath(),
                 "--" + CopyNumberStandardArgument.DENOISED_COPY_RATIOS_FILE_LONG_NAME, DENOISED_COPY_RATIOS_FILE.getAbsolutePath(),
-                "-" + StandardArgumentDefinitions.SEQUENCE_DICTIONARY_NAME, SEQUENCE_DICTIONARY_WITH_NO_CONTIGS_ABOVE_MINIMUM_LENGTH_FILE.getAbsolutePath(),
+                "--" + StandardArgumentDefinitions.SEQUENCE_DICTIONARY_NAME, SEQUENCE_DICTIONARY_WITH_NO_CONTIGS_ABOVE_MINIMUM_LENGTH_FILE.getAbsolutePath(),
```

Comment:
Why does this StandardArgumentDefinitions us "--" but the next one for OUTPUT_SHORT_NAME uses only "-".

Reply:
Short names use `-`, long names use `--`.  It just happens that the short and long names for sequence dictionary are the same for the CNV plotting tools, but this line of code would've broken if that was ever changed.